data.json

```diff
@@ -995,13 +995,11 @@
     "imageUrl": "https://user-images.githubusercontent.com/10162710/120894576-6ff8f200-c619-11eb-9ee5-5078912d5550.png",
```

Comment:
Still Changes are required please fix it.

Reply:
Hey @skjha1 instead of just asking them to fix things, please tell them what they need to fix as well

Reply:
Hey @Leticiarosario [Refer this](https://github.com/EddieHubCommunity/awesome-github-profiles#to-add-your-github-profile)

Reply:
@skjha1 I didn't understand what you were trying to explain about changes. Could you please explain? :slightly_smiling_face:

Reply:
There was nothing to understand I said refer readme, everything is mentioned there.

Reply:
I asked because they already did, I don't see anything that you might be referring to which needs to be changed. That's why asked if you could explain :slightly_smiling_face: